### DISCUSSION TITLE
Running MoonRay on an Apple Silicon Mac

### DISCUSSION THREAD

**Discussion by fourcandy** [dreamworksanimation/openmoonray#86] [Q&A]
Hi. Is there any way I can use MoonRay on an M1 Mac? I tried building it in docker but it got stuck in some step (can't remember which one).

**Comment by toshi-kato** [comment]
Thank you for your interest in Apple Silicon version of OpenMoonRay.
The MoonRay development team is using the Linux platform due to support DreamWorks studio production. We would love to see the Apple Silicon version supported through community contributions. 
FYI, The community already started to discuss similar [topics](https://github.com/dreamworksanimation/openmoonray/issues/37).
